### PR TITLE
k8s AWS EBS static persistent volumes

### PR DESCRIPTION
**Pull request by vkhromov**

### CODE REVIEW

**Comment by solarkennedy** [comment]
Can you rebase on master?

File: paasta_tools/kubernetes_tools.py [hunk 1]
```diff
@@ -269,6 +272,22 @@ def get_sanitised_volume_name(self, volume_name: str) -> str:
...
+    def get_aws_ebs_volume_name(self, aws_ebs_volume: AwsEbsVolume) -> str:
+        return self.get_sanitised_volume_name(
+            'aws-ebs--{name}{partition}'.format(
+                name=aws_ebs_volume['volume_id'],
+                partition=aws_ebs_volume.get('partition', ''),
+            ),
+        )
+
+    def read_only_mode(self, d: VolumeWithMode) -> bool:
+        return True if d.get('mode', 'RO') == 'RO' else False
```
Comment (Rob-Johnson) [inline paasta_tools/kubernetes_tools.py]: ``return d.get('mode', 'RO') == 'RO'``

File: paasta_tools/kubernetes_tools.py [hunk 2]
```diff
@@ -368,32 +388,72 @@ def get_kubernetes_containers(
...
 
-    def get_volume_mounts(self, volumes: Sequence[DockerVolume]) -> Sequence[V1VolumeMount]:
+    def get_volume_mounts(
+        self,
+        docker_volumes: Sequence[DockerVolume],
+        aws_ebs_volumes: Sequence[AwsEbsVolume],
+    ) -> Sequence[V1VolumeMount]:
         volume_mounts = []
-        for volume in volumes:
+        for docker_volume in docker_volumes:
```
Comment (Rob-Johnson) [inline paasta_tools/kubernetes_tools.py]: this function could be DRY'd a little - can you:

- rename DockerVolume.containerPath to DockerVolume.container_path
- move logic for how to rename ``DockerVolume`` vs ``AwsEbsVolume`` to  ``get_aws_ebs_volume_name``
- reduce this function to 'just' be:
```
[V1VolumeMount(
     mount_path=aws_ebs_volume['container_path'],
     name=self.get_aws_ebs_volume_name(aws_ebs_volume),
     read_only=self.read_only_mode(aws_ebs_volume),
) for volume in aws_ebs_volumes + docker_volumes]
```

Comment (vkhromov) [reply paasta_tools/kubernetes_tools.py]: `containerPath` is actually a name of the property in our configs from `srv-configs`, we cannot rename that easily.

Comment (vkhromov) [reply paasta_tools/kubernetes_tools.py]: Also `get_aws_ebs_volume_name()` can't be used for `DockerVolume` instances.

File: paasta_tools/kubernetes_tools.py (hunk 1 above)
Comment (vkhromov) [reply paasta_tools/kubernetes_tools.py]: done

**Pull request merged**